PS5 NOR Modifier/NORInfo.cs

```diff
@@ -0,0 +1,12 @@
+namespace PS5_NOR_Modifier;
+
+public class NORData
+{
+    public string? OffsetOne; // TODO: These two need better names
+    public string? OffsetTwo;
+    public string? WiFiMAC;
+    public string? LANMAC;
+    public string? Serial;
+    public string? Variant;
+    public string? MoboSerial;
```

Comment:
These should remain serialized as `byte[]`s, or even better the entire NOR should be read into memory and wrapped by a class with functions for extracting this information as `byte[]`s/`ReadOnlySpan<byte>`s.